tests/network/general/test_bridge_marker.py

```diff
@@ -127,11 +127,12 @@ def non_homogenous_bridges(worker_node1, worker_node2):
             yield bridgemarker2_ncp, bridgemarker3_ncp
 
 
-def _assert_failure_reason_is_bridge_missing(pod, bridge):
-    cond = pod.instance.status.conditions[0]
-    missing_resource = bridge.resource_name
-    assert cond.reason == "Unschedulable"
-    assert f"Insufficient {missing_resource}" in cond.message
+def _assert_failure_reason_is_bridge_missing(vmi, bridge):
+    for cond in vmi.instance.status.conditions:
+        if cond.reason == "Unschedulable":
```

Comment:
while this works
it will be more accurate please to search for the cond of `type: PodScheduled`, assert it has `status: "False"`
and then rest of the logic here about reason / message

Reply:
Thanks for the comment @oshoval , you suggestion indeed produces a more correct check.